requirements_dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
@BibianaC Sorry last thing - please could you re-compile using Python 3.8 instead of Python 3.12 , so it's the same as `requirements.txt` https://github.com/ThreeSixtyGiving/grantnav/blob/live/requirements.txt#L2
 ... we'll get around to upgrading python one day soon :)